### ISSUE TITLE
PTN000280591

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#3939]
* **PTHR ID & PTN node:**

GO:0043985 | histone H4-R3 methylation | IBA with locus:2161967 , PTN000280591

* **Sequences with problematic annotation (ID + gene/protein name):**
https://www.pombase.org/gene/SPAC25H1.02

* **Type of Issue: Erroneous source or erroneous propagation, or other issue**

Erroneous source? the is a demethylase, not a methylase?

* Add the label 'high priority' if needed. Generally high priority issues affect a lot of proteins, and annotations are incorrect (as opposed to just imprecise).

**Comment by sjm41** [comment]
Other problem here is that "histone H3-methyl-arginine-3 demethylase activity" is being added to JMJD4 proteins (e.g. fly Q9VJ97, human Q9H9V9), which are not histone demethylases.

**Comment by ValWood** [comment]
@sjm41 are you sure? I thought the JmjC domain was characteristic a histone demethylase?

@colinlog may know...

**Comment by ValWood** [comment]
https://pubmed.ncbi.nlm.nih.gov/26404469/
but maybe they ate not all specific for histones?

**Comment by colinlog** [comment]
I could not access the book chapter Val. They mention a JMJ-C subfamily that is not likely to be a histone demethylase.

Histones are very abundant and therefore their modified forms are easier to detect and study than other chromatin proteins. Nevertheless, I know of many lysine and arginine methylation sites in chromatin proteins. Therefore it makes sense that demethylases act on them. To my knowledge no-one has excluded JMJ-C from demethylating other lysines. I view this as 'signal reinforcement', whereby both the demthylated histone and the associated demethylated putative chromatin remodelling complex subunit act together. That stops upon their quasi-simultaneous demethylation. 

The abstract of PMID: 26152513 observes the same extra-JMJ-C sub-family and adds that at least one JMJ-C sub-family has another activity, namely histone H4 hydroxylation. PMID: 19574390, 29176719, say that "Jumonji domain containing 6 (JMJD6),2 which possesses high binding affinity to single-stranded RNA, is reported to hydroxylate lysyl residues of an RNA splicing factor, U2AF65". JMJD6 is an arginine and a lysine de-metylase. 

 The PMID: 26152513 classification spans to plants and appears to be consistent with JMJ-C cannonical sub-families and their histone substrate conservation through to man and yeast for some sub-families at least. So, it would be nice to see if Panther PAINTing the JMJ-C (with @pgaudet Pascale) would help restrain the erroneous transfer to any of the Drosophila JMJ-C domain proteins *whilst still hitting the right ones when expressed as histone code demethylases*?

**Comment by sjm41** [comment]
@ValWood Yes, I'm sure.  The human and fly examples above (and other JMJD members) are characterized as 2-oxoglutarate-dependent dioxygenases, but they are not demethylases.

From https://pubmed.ncbi.nlm.nih.gov/31034925/:
Jumonji C domain-containing (JMJD) proteins are mostly epigenetic regulators that demethylate histones. However, a hitherto neglected subfamily of JMJD proteins, evolutionarily distant and characterized by their relatively small molecular weight, exerts different functions by hydroxylating proteins and RNA. Recently, unsuspected proteolytic and tyrosine kinase activities were also ascribed to some of these small JMJD proteins, further increasing their enzymatic versatility. Here, we discuss the ten human small JMJD proteins (HIF1AN, HSPBAP1, JMJD4, JMJD5, JMJD6, JMJD7, JMJD8, RIOX1, RIOX2, TYW5) and their diverse physiological functions.

**Comment by ValWood** [comment]
right, that subgroup already appear to be a different panther family, so it might be easy to fix...

http://www.pantherdb.org/panther/family.do?clsAccession=PTHR12480

**Comment by marcfeuermann** [comment]
I've removed both methylase (an annotation error by TAIR) and demethylase activities, as suggested.

**Comment by sjm41** [comment]
Thanks Marc.
